packages/neon/neon/lib/src/pages/search.dart

```diff
@@ -0,0 +1,88 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:neon/l10n/localizations.dart';
+import 'package:neon/neon.dart';
+import 'package:nextcloud/nextcloud.dart';
+import 'package:provider/provider.dart';
+
+class SearchPage extends StatefulWidget {
+  const SearchPage({
+    super.key,
+  });
+
+  @override
+  State<SearchPage> createState() => _SearchPageState();
+}
+
+class _SearchPageState extends State<SearchPage> {
+  late final UnifiedSearchBloc bloc;
+
+  @override
+  void initState() {
+    super.initState();
+
+    bloc = Provider.of<AccountsBloc>(context, listen: false).activeAppsBloc.unifiedSearchBloc;
+  }
+
+  @override
+  Widget build(final BuildContext context) => Scaffold(
+        appBar: AppBar(
+          title: Text(AppLocalizations.of(context).search),
+        ),
+        body: ResultBuilder<Iterable<NextcloudCoreUnifiedSearchProvider>>(
+          stream: bloc.providers,
+          builder: (final context, final providers) => NeonListView(
+            items: providers.data,
+            isLoading: providers.loading,
+            error: providers.error,
+            onRefresh: bloc.refresh,
+            builder: _buildProvider,
```

Comment:
Yep, that would be really good. In this PR I noticed again how not great the NeonListView is right now (not even performance wise but just how you need to use it).

Reply:
I wanted to add it now, but it doesn't make much sense because of the sorting that is going on. There is only on occurrences where we directly pass the result to the list view.